README.md

```diff
@@ -115,3 +115,55 @@ in normal/insert mode:
 * `F5` to execute a block of code scvim will attempt to find the outermost bracket
 * `F6` to execute the current line of code
 * `F12` is a hard stop
+
+
+Using scvim on command-line only systems: `tmux` (e.g. Raspbian light)
+----------------------------------------------------------------------
+
+When sclang is started from scvim, another terminal is opened in which sclang
+output is posted. In a command line only interface this is not possible out of
+the box. Therefore scvim makes use of `tmux` to provide multiple frames in a
+single terminal window. Tmux is provided by all Linux distributions, as well as
+by Homebrew on the Mac. While tmux offers many configuration options, basic use
+just requires to run `tmux` before `vim`. Then sclang output will appear in a
+split window at the bottom of the screen.
+
+*NOTE for Linux*: at the time of this writing (3.9dev) there is a bug in sclang
```

Comment:
To clarify—it's the clearest way to point to a specific point in the history of the repo. The bug might be fixed still in 3.9-dev, so that statement that the bug is in 3.9-dev may become inaccurate over time. Or at least refer to the issue number so people don't have to dig it up themselves. :)